### DISCUSSION TITLE
Another Baby Buddy integration: MatrixPortal case

### DISCUSSION THREAD

**Discussion by skjdghsdjgsdj** [babybuddy/babybuddy#920] [Show and tell]
Hardware: https://github.com/skjdghsdjgsdj/matrixportal-case
Software: https://github.com/skjdghsdjgsdj/matrixportal-babybuddy

I have two of these in my home and they've proven very useful to check a child's status at a glance, like "why are they cranky...oh because they haven't slept for 5 hours" and that sort of thing. My other big project [BabyPod](https://github.com/skjdghsdjgsdj/babypod-hardware) creates timer names that are compatible with this too.

Feel free to add to the "Baby Buddy on the Web" section of `README.md` if you like.

**Comment by cdubz** [comment]
I'll definitely add to BBotW. And I promise I'll get back to MQTT!
